package Lab_Excercises;

public class ThrowsException {

	public static void main(String[] args)throws ArithmeticException {
		System.out.println("Inside main"); 
		 int I = 0; 
		 int j = 40/I; 
		 System.out.println("this statement is not printed"); 

	}

}
